tensilelite/Tensile/Tests/common/gemm/gemm_ck_gfx942.yaml

```diff
@@ -46,21 +46,24 @@ BenchmarkProblems:
       CustomKernels:
         [
           "Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_DTVA",
-          "Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_WSGRB2_DTVA"
+          "Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x256x64_MI16x16x1_SN_K1_MIWT4_16_WSGRB2_DTVA",
+          "Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x224x64_MI16x16x1_SN_K1_MIWT4_14_DTVA",
+          "Custom_Cijk_Alik_Bljk_BBS_BH_Bias_AS_SAV_UserArgs_MT256x224x64_MI16x16x1_SN_K1_MIWT4_14_WSGRB2_DTVA"
         ]
       InitialSolutionParameters:
       BenchmarkCommonParameters:
         - KernelLanguage: ["Assembly"]
       ForkParameters:
         - MatrixInstruction:
           - [16, 16,16, 1,  1,   4,16,  4,1 ] # 256x256 (Occupancy 1)
+          - [16, 16,16, 1,  1,   4,14,  4,1 ] # 256x224
 
         - AssertFree0ElementMultiple: [1]
         - AssertFree1ElementMultiple: [1]
         # - AssertSummationElementMultiple: [64]
         - PrefetchGlobalRead: [2]
         - PrefetchLocalRead: [1]
-        - DepthU: [64]
+        - DepthU: [128]
```

Comment:
DU of kernel name is 64. Any missing?

Reply:
Fixed it.
But actually this yaml is only for testing custom kernels.
No other kernels should be generated successfully.
So this value won't influence the test results.

Reply:
In this case you may remove all of the ForkParameters.

Reply:
Got it! Done.